test/retiarii_test/lightning/test_oneshot.py

```diff
@@ -0,0 +1,142 @@
+import torch
```

Comment:
You put the test in the wrong place. It should be put into `test/ut/retiarii`.
`test/retiarii_test` is an unmonitored folder that is never scanned.